src/ID3Definitions.js

```diff
@@ -209,8 +209,79 @@ const ENCODINGS = [
     'ISO-8859-1', 'UTF-16', 'UTF-16BE', 'utf8'
 ]
 
+/**
+ * Documented constants used in tag frames.
+ *
+ * @see {@link https://id3.org/} for more information.
+ */
+const Constants = {
```

Comment:
I have already defined constants for `ETCO` and `COMR` frames but did not realise they were not supported yet, so I did not include the definitions.
Also definitions for `TFLT` and `TMED` are a bit special, I did not include them in this PR.

Reply:
How about you commit the ETCO and COMR constants and I will implement them tomorrow? Then everything would be in place

Reply:
I pushed a commit with them.